### PR TITLE
Refactor lodash methods to use handlebars-utils instead

### PR DESCRIPTION
**Pull request by bookernath**
## What? Why?
Swapping out lodash methods with handlebars-utils, which is a little nicer for handlebars.

## How was it tested?
Unit tests still pass.

----

cc @bigcommerce/storefront-team

### CODE REVIEW

**Comment by bookernath** [comment]
In terms of performance, this does seem to run the test suite a few ms faster than master.

**Comment by ncheikh** [comment]
Unless I am mistaken some of these files are the same modified in: https://github.com/bigcommerce/paper-handlebars/pull/59

Just an FYI: @bc-williamkwon

File: helpers/contains.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const _ = require('lodash');
 const factory = () => {
     return function() {
         var args = Array.prototype.slice.call(arguments, 0, -1),
-            options = _.last(arguments),
+            options = arguments[arguments.length - 1],
```
Comment (ncheikh) [inline helpers/contains.js]: 👍 appreciate the removal of lodash here for this check

**Comment by bookernath** [comment]
@ncheikh ♻️

**Pull request merged**